packages/starlight-theme-ensnode/package.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "starlight-theme-ensnode",
+  "version": "0.1.0",
+  "license": "MIT",
+  "description": "Starlight theme for ENSNode inspired by the official ENS branding",
+  "author": "Jamie Barton <jamie@notrab.dev>",
+  "type": "module",
+  "exports": {
+    ".": "./index.ts",
+    "./styles": "./styles.css",
+    "./package.json": "./package.json",
+    "./overrides/ThemeSelect.astro": "./overrides/ThemeSelect.astro"
+  },
+  "peerDependencies": {
+    "@astrojs/starlight": ">=0.32.0"
+  },
+  "engines": {
+    "node": ">=18"
+  },
+  "packageManager": "pnpm@9.0.4",
```

Let's upgrade these settings to follow changes from #484:

```suggestion
  "engines": {
    "node": ">=22.14.0"
  },
  "packageManager": "pnpm@10.7.1",
```